### PR TITLE
drop duplicate data points from other traces

### PR DESCRIPTION
**Pull request by SteffenHeu**
Apparently it is not guaranteed that data points for chromatograms are unique or sorted. Occurred in bruker chromatography files. I put the same changes into the mzml import to be safe, as the bruker files may be converted to mzml.

### CODE REVIEW

File: mzmine-community/src/main/java/io/github/mzmine/modules/io/import_rawdata_bruker_uv/BrukerUvReader.java [hunk 1]
```diff
@@ -316,14 +319,39 @@ private OtherTimeSeries loadTimeSeriesData(MemoryMapStorage storage,
...
+   */
+  private @NotNull List<DataPoint> dropDuplicateRtDataPoints(Trace trace,
+      TFloatArrayList intensitiesList, TDoubleArrayList timesList) {
+    List<DataPoint> dps = new ArrayList<>();
+    for (int i = 0; i < intensitiesList.size(); i++) {
+      final SimpleDataPoint dp = new SimpleDataPoint(timesList.get(i), intensitiesList.get(i));
+      dps.add(dp);
+    }
+
+    // intermediate filtering to remove duplicates from traces. Why is this even necessary?
```
Comment (robinschmid) [inline mzmine-community/src/main/java/io/github/mzmine/modules/io/import_rawdata_bruker_uv/BrukerUvReader.java:343]: maybe the detector is super fast and the rounding is too high? Not sure.

Comment (SteffenHeu) [reply mzmine-community/src/main/java/io/github/mzmine/modules/io/import_rawdata_bruker_uv/BrukerUvReader.java:343]: No it's repeated values

File: mzmine-community/src/main/java/io/github/mzmine/modules/io/import_rawdata_mzml/ConversionUtils.java [hunk 2]
```diff
@@ -436,6 +444,28 @@ public static List<OtherDataFile> convertOtherTraces(RawDataFile file,
...
+      MzMLChromatogram chrom) {
+    final List<DataPoint> dps = new ArrayList<>();
+    final float[] rts = chrom.getRetentionTimes();
+    final double[] intensities = chrom.getIntensities();
+    for (int i = 0; i < chrom.getNumberOfDataPoints(); i++) {
+      dps.add(new SimpleDataPoint(rts[i], intensities[i]));
+    }
+    dps.sort(DataPointSorter.DEFAULT_MZ_ASCENDING);
+    final int before = dps.size();
+    CollectionUtils.dropDuplicatesRetainOrder(dps);
```
Comment (robinschmid) [inline mzmine-community/src/main/java/io/github/mzmine/modules/io/import_rawdata_mzml/ConversionUtils.java:460]: same here - drop solely based on RT?

Comment (SteffenHeu) [reply mzmine-community/src/main/java/io/github/mzmine/modules/io/import_rawdata_mzml/ConversionUtils.java:460]: No it's repeated value pairs

**Pull request merged**